cmd/controller/controller.go

```diff
@@ -156,6 +156,11 @@ func AddConfigFlags(cmd *cobra.Command) {
 		false,
 		"Permits PodSpecPatch to modify the command or args fields of stack-provided containers. See the warning in the README before enabling this option",
 	)
+	cmd.Flags().Bool(
+		"experimental-job-reservation-support",
+		false,
+		"Please contact us before enabling this flag. This feature enables job reservation support: Better job observability, scalable job fetching!",
```

Comment:
Ah... I had meant to remind you to update this text to be consistent with the JSON schema.

Reply:
Oops quick follow up: https://github.com/buildkite/agent-stack-k8s/pull/647